worker-policy/src/main/java/com/github/cafdataprocessing/worker/policy/ExecuteTaskData.java

```diff
@@ -51,6 +51,7 @@
  * Main logic class for handling executing against a task data object.
  */
 public class ExecuteTaskData {
+    private static final Object syncKey = new Object();
```

Comment:
Rename variable to `registeredUsersLock` to indicate its link with `registeredUsers`.